tests/modules/net_lb_ext/examples/dual_stack.yaml

```diff
@@ -215,7 +215,6 @@ values:
     description: Terraform managed.
     enable_cdn: null
     failover_policy: []
-    iap: []
```

Comment:
This error kept popping up for me. If this is not the right way to address this, please let me know.

```
================================================================= short test summary info =================================================================
FAILED tests/examples/test_plan.py::test_example[terraform:modules/net-lb-ext:Referencing existing MIGs:1] - AssertionError: tests/modules/net_lb_ext/examples/migs.yaml: .module.nlb.google_compute_region_backend_service.default.iap is not a valid address in t...
FAILED tests/examples/test_plan.py::test_example[terraform:modules/net-lb-ext:Externally managed instances:1] - AssertionError: tests/modules/net_lb_ext/examples/ext_migs.yaml: .module.nlb.google_compute_region_backend_service.default.iap is not a valid address ...
FAILED tests/examples/test_plan.py::test_example[terraform:modules/net-lb-ext:Multiple forwarding rules:1] - AssertionError: tests/modules/net_lb_ext/examples/fwd_rules.yaml: .module.nlb.google_compute_region_backend_service.default.iap is not a valid address...
FAILED tests/examples/test_plan.py::test_example[terraform:modules/net-lb-ext:Dual stack (IPv4 and IPv6):1] - AssertionError: tests/modules/net_lb_ext/examples/dual_stack.yaml: .module.nlb.google_compute_region_backend_service.default.iap is not a valid addres...
FAILED tests/examples/test_plan.py::test_example[terraform:modules/net-lb-ext:End to end example:1] - AssertionError: tests/modules/net_lb_ext/examples/e2e.yaml: .module.nlb.google_compute_region_backend_service.default.iap is not a valid address in th...
====================================================== 5 failed, 978 deselected in 113.49s (0:01:53) ======================================================
```